### ISSUE TITLE
 Write a script called build.sh that will create the thyra-plugin-wallet Binary, taking in two arguments: the port number and the path to the plugin.

### ISSUE THREAD

**Issue by B-Naoufal** [massalabs/thyra-plugin-wallet#11]

**Comment by SlnPons** [comment]
@B-Naoufal can you add more details about what's the purpose of this issue? 
To be able to install the plug-in easily, is that correct? Cheers

**Comment by B-Naoufal** [comment]
@SlnPons the goal is to have a build file build.sh, which can be called by the CI in this repository 
thyra-plugin-wallet and produces in input a binary (executable file ), thyra will after automatically download this binary file, and install the plugin wallet:  

Any Pull Request in [thyra-plugin-wallet](https://github.com/massalabs/thyra-plugin-wallet) ==> Merge ==> CI ==> call build.sh ==> Produce thyra-plugin-wallet Binary.  
So When you launch thyra:  
Call Plugin Manager ==> Download the last Produced thyra-plugin-wallet Binary By thyra-plugin-wallet CI

**Comment by SlnPons** [comment]
Hey @B-Naoufal I failed at accessing the plug-in wallet as described above. 
Here is what I did: 
1. Download the binary on this page: https://github.com/massalabs/thyra-plugin-wallet/actions/runs/3764513478
2. Execute the binary using the following cmd on my terminal `wallet-plugin chmod +x thyra-plugin-wallet`
3. Try to access the wallet plugin on port 8080 as described on my terminal (see screenshot below)


![Image](https://user-images.githubusercontent.com/109611779/209657922-c6317dfa-c795-4e04-91a3-7e869152fd1b.png)


4. Get a 404 error
